modules/game/src/main/GameRepo.scala

```diff
@@ -196,7 +196,7 @@ final class GameRepo(val coll: Coll)(implicit ec: scala.concurrent.ExecutionCont
 
   // Use Env.round.proxy.urgentGames to get in-heap states!
   def urgentPovsUnsorted(user: User): Fu[List[Pov]] =
-    coll.list[Game](Query nowPlaying user.id, Game.maxPlayingRealtime) dmap {
+    coll.list[Game](Query nowPlaying user.id, Game.maxPlaying + 5) dmap {
```

Comment:
I do not understand why you did add this +5 here ?
What does it represent ?

Reply:
This is what lichess have done, I assume it makes sure you get all the games, as perhaps you can create a few more than the 200 limit in some edge case scenarios.....